lib/builder.dart

```diff
@@ -12,9 +12,24 @@ import 'dart:async';
 import 'package:build/build.dart';
 import 'package:pubspec_parse/pubspec_parse.dart';
 
-Builder buildVersion([BuilderOptions options]) => _VersionBuilder();
+Builder buildVersion([BuilderOptions options]) => _VersionBuilder(options);
+
+// The top level keys supported for the `options` config for the [_VersionBuilder].
+const String _fieldName = 'field_name';
+const String _targetPath = 'target_path';
 
 class _VersionBuilder implements Builder {
+  _VersionBuilder([BuilderOptions options])
+      : fieldName = options != null && options.config.containsKey(_fieldName)
```

Comment:
Maybe create a static helper for these guys?

We want to do `option.config[key] as String` instead of `toString()`. We don't want to let an array/map/bool etc get through with a toString call.